scripts/generate_tf_guides.py

```diff
@@ -22,6 +29,26 @@
         "source_name": "making_new_layers_and_models_via_subclassing",
         "target_name": "custom_layers_and_models",
     },
+    {
+        "title": "Save and load Keras models",
+        "source_name": "serialization_and_saving",
+        "target_name": "save_and_serialize",
+    },
+    {
+        "title": "Working with preprocessing layers",
+        "source_name": "preprocessing_layers",
+        "target_name": "preprocessing_layers",
+    },
+    {
+        "title": "Customizing what happens in `fit()`",
```

Comment:
This might display weird in the left nav. Maybe "Customize what happens in Model.fit"

Reply:
Done